src/ensembl/webpack/webpack.config.prod.js

```diff
@@ -24,7 +24,7 @@ const moduleRules = [
   // image loader should compress the images
   // then file loader takes over to copy the images into the dist folder
   {
-    test: /static\/img\/.*\.(gif|png|jpe?g)$/i,
+    test: /.*\.(gif|png|jpe?g)$/i,
     use: [
       {
         loader: 'file-loader',
```

Comment:
Right now, this preserves the original folder structure when copying files to `dist` folder (so that images in `src/shared/error-screen/images` folder end up in `dist/src/shared/error-screen/images`). Perhaps we might use the `outputPath` option of fileLoader to put all image files in something like `dist/static/images` (though I never tried this).

Reply:
We will need to do something like that for images that come from the `src` directory. However, I think we will need to maintain the path of the `static` folder images as they are divided into separate directories.

Reply:
Updated the config file so that now all images go into the `dist/images` folder (will change it to `dist/static/images` in the `cicd-test` branch, but that's for another PR).

Sample output (3 files from `src/shared/error-screen`, and one from `static/images/...`):

![image](https://user-images.githubusercontent.com/6834224/59110139-56dbde80-8936-11e9-88b1-59b84345ac5b.png)